frontend/src/metabase/internal/lib/components-webpack.js

```diff
@@ -22,4 +24,16 @@ const components = [
   ...getComponents(containersReq),
 ];
 
+// provide some stats on the total vs total documented components
+const documented = getComponents(componentsReq).length;
+
+// get everything and then subtract documented
+const total = getComponents(allComponentsReq).length - documented;
```

Comment:
I don't think we should subtract the `documented` count. If we do, it's not really a total, right?

Is the issue that we're double counting documented components in `allComponentsReq` because we import both the normal file and the `.info` one?